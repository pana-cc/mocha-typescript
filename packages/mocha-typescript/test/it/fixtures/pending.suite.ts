import { pending, suite, test } from "../../../index";

@suite @pending class Suite1 {
    @test public test1() {}
}
@suite class Suite2 {
    @test public test1() {}
}
