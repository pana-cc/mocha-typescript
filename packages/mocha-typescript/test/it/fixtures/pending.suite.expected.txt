
  Suite1
    - test1

  Suite2
    âœ“ test1

  1 passing
  1 pending
